deployment/GI03FLMB_Deploy.csv

```diff
@@ -129,3 +129,29 @@ AR30-03,,,,GI03FLMB-RIS01-00-SIOENG000,5,1,2018-06-11T14:35:00,,CGMGI-03FLMB-000
 AR30-03,,,,GI03FLMB-RIS01-03-DOSTAD000,5,1,2018-06-11T14:35:00,,CGMGI-03FLMB-00005,,CGINS-DOSTAD-00394,59.71427,-39.31329,,30,2829,
 AR30-03,,,,GI03FLMB-RIS01-04-PHSENF000,5,1,2018-06-11T14:35:00,,CGMGI-03FLMB-00005,,CGINS-PHSENF-P0181,59.71427,-39.31329,,30,2829,
 AR30-03,,,,GI03FLMB-RIS01-05-FLORTD000,5,1,2018-06-11T14:35:00,,CGMGI-03FLMB-00005,,CGINS-FLORTD-01320,59.71427,-39.31329,,30,2829,
+AR35-05,,,,GI03FLMB-RI000-00-CTDMOH000,6,1,,,CGMGI-03FLMB-00006,,CGINS-CTDMOH-13454,,,,1830,2812,
+AR35-05,,,,GI03FLMB-RI000-00-CTDMOH100,6,1,,,CGMGI-03FLMB-00006,,CGINS-CTDMOH-13457,,,,2730,2812,
+AR35-05,,,,GI03FLMB-RI000-00-CTDMOH400,6,1,,,CGMGI-03FLMB-00006,,CGINS-CTDMOH-13456,,,,2430,2812,
+AR35-05,,,,GI03FLMB-RI000-00-CTDMOH700,6,1,,,CGMGI-03FLMB-00006,,CGINS-CTDMOH-13455,,,,2130,2812,
+AR35-05,,,,GI03FLMB-RI000-00-VELPTB000,6,1,,,CGMGI-03FLMB-00006,,CGINS-VELPTB-12236,,,,1830,2812,
+AR35-05,,,,GI03FLMB-RI000-00-VELPTB100,6,1,,,CGMGI-03FLMB-00006,,CGINS-VELPTB-12673,,,,2730,2812,
+AR35-05,,,,GI03FLMB-RI000-00-VELPTB400,6,1,,,CGMGI-03FLMB-00006,,CGINS-VELPTB-12663,,,,2430,2812,
+AR35-05,,,,GI03FLMB-RI000-00-VELPTB700,6,1,,,CGMGI-03FLMB-00006,,CGINS-VELPTB-12459,,,,2130,2812,
+AR35-05,,,,GI03FLMB-RIM01-00-SIOENG000,6,1,,,CGMGI-03FLMB-00006,,CGCON-SIOCON-14874,,,,1507,2812,
+AR35-05,,,,GI03FLMB-RIM01-02-ADCPSL007,6,1,,,CGMGI-03FLMB-00006,,CGINS-ADCPSL-23381,,,,500,2812,
+AR35-05,,,,GI03FLMB-RIM01-02-CTDMOG060,6,1,,,CGMGI-03FLMB-00006,,CGINS-CTDMOG-11594,,,,30,2812,
+AR35-05,,,,GI03FLMB-RIM01-02-CTDMOG061,6,1,,,CGMGI-03FLMB-00006,,CGINS-CTDMOG-12609,,,,40,2812,
+AR35-05,,,,GI03FLMB-RIM01-02-CTDMOG062,6,1,,,CGMGI-03FLMB-00006,,CGINS-CTDMOG-11645,,,,60,2812,
+AR35-05,,,,GI03FLMB-RIM01-02-CTDMOG063,6,1,,,CGMGI-03FLMB-00006,,CGINS-CTDMOG-11604,,,,90,2812,
+AR35-05,,,,GI03FLMB-RIM01-02-CTDMOG064,6,1,,,CGMGI-03FLMB-00006,,CGINS-CTDMOG-11534,,,,130,2812,
+AR35-05,,,,GI03FLMB-RIM01-02-CTDMOG065,6,1,,,CGMGI-03FLMB-00006,,CGINS-CTDMOG-11606,,,,180,2812,
+AR35-05,,,,GI03FLMB-RIM01-02-CTDMOG066,6,1,,,CGMGI-03FLMB-00006,,CGINS-CTDMOG-11608,,,,250,2812,
+AR35-05,,,,GI03FLMB-RIM01-02-CTDMOG067,6,1,,,CGMGI-03FLMB-00006,,CGINS-CTDMOG-11603,,,,350,2812,
+AR35-05,,,,GI03FLMB-RIM01-02-CTDMOG068,6,1,,,CGMGI-03FLMB-00006,,CGINS-CTDMOG-12190,,,,500,2812,
+AR35-05,,,,GI03FLMB-RIM01-02-CTDMOH069,6,1,,,CGMGI-03FLMB-00006,,CGINS-CTDMOH-12227,,,,750,2812,
+AR35-05,,,,GI03FLMB-RIM01-02-CTDMOH070,6,1,,,CGMGI-03FLMB-00006,,CGINS-CTDMOH-11703,,,,1000,2812,
+AR35-05,,,,GI03FLMB-RIM01-02-CTDMOH071,6,1,,,CGMGI-03FLMB-00006,,CGINS-CTDMOH-12219,,,,1500,2812,
+AR35-05,,,,GI03FLMB-RIS01-00-SIOENG000,6,1,,,CGMGI-03FLMB-00006,,CGCON-SIOCON-12372,,,,30,2812,
+AR35-05,,,,GI03FLMB-RIS01-03-DOSTAD000,6,1,,,CGMGI-03FLMB-00006,,CGINS-DOSTAD-00224,,,,30,2812,
+AR35-05,,,,GI03FLMB-RIS01-04-PHSENF000,6,1,,,CGMGI-03FLMB-00006,,CGINS-PHSENF-P0183,,,,30,2812,
+AR35-05,,,,GI03FLMB-RIS01-05-FLORTD000,6,1,,,CGMGI-03FLMB-00006,,CGINS-FLORTD-01221,,,,30,2812,

```

Comment:
Looks good. Water depth will need to be updated after deployment, and deployment depth of the OSNAP instruments which should be 100, 400, 700, and 1000 less than the water depth.